src/node.rs

```diff
@@ -123,6 +131,9 @@ pub enum NodeName {
 
     /// Name separated by colons, e.g. `<div on:click={foo} />`
     Colon(Punctuated<Ident, Colon>),
+
+    /// Name specified by arbitrary rust code in braced `{}` blocks
```

Comment:
Since syn-rsx isn't aware whether the block actually specifies a name after all, I think it'd be enough to doc comment like the NodeType: "Arbitrary rust code in braced `{}` blocks"